contents/docs/ai-engineering/_snippets/anthropic.mdx

```diff
@@ -44,6 +73,27 @@ response = client.messages.create(
 print(response.content[0].text)
 ```
 
+```typescript
+const response = await client.messages.create({
+  model: "claude-3-opus-20240229",
+  messages: [
+    {
+      role: "user",
+      content: "Tell me a fun fact about hedgehogs"
+    }
+  ],
+  posthogDistinctId: "user_123", // optional
+  posthogTraceId: "trace_123", // optional
+  posthogProperties: { conversationId: "abc123", paid: true }, // optional
+  posthogGroups: { company: "company_id_in_your_db" }, // optional
+  posthogPrivacyMode: false // optional
+})
+
+console.log(response.content[0].text)
```

Comment:
Also need to do the phClient.shutdown() on TS.

It seems to be something people trip up on as a lot are using in serverless functions